tests/Admin/Extension/CreateSnapshotAdminExtensionTest.php

```diff
@@ -95,4 +95,24 @@ public function testPostPersistOnBlock(): void
         $extension = new CreateSnapshotAdminExtension($backend);
         $extension->postPersist($admin, $block);
     }
+
+    public function testPostRemoveOnBlock(): void
+    {
+        $page = $this->createMock(PageInterface::class);
+        $page->expects($this->once())->method('getId')->willReturn(42);
+
+        $block = $this->createMock(PageBlockInterface::class);
+        $block->expects($this->once())->method('getPage')->willReturn($page);
+
+        $admin = $this->createMock(AdminInterface::class);
```

Comment:
```suggestion
        $admin = $this->createStub(AdminInterface::class);
```

Reply:
Are you sure? I copied from the other methods where `createMock` is used.

Reply:
Yes, the other ones needs to be changed.

The difference is that a Stub doesn't have any expectation, just return things, only these methods are available: https://github.com/sebastianbergmann/phpunit/blob/75b9ac8f448d929ef450c8d4221aa7206a62dc66/src/Framework/MockObject/Builder/InvocationStubber.php

In this case `$admin` is used like that, as a stub, there are no calls to `->expects()` nor `->with()`, if you are interested, [here you can read more about that](https://github.com/sebastianbergmann/phpunit/issues/3120).